tilequeue/command.py

```diff
@@ -377,6 +377,11 @@ def parse_layer_data(query_cfg, template_path, reload_templates, cfg_path):
         else:
             template = environment.get_template(template_name)
             query_generator = JinjaQueryGenerator(template, start_zoom)
+            layer_config.get('area-inclusion-threshold')
+            if layer_config is None:
+                area_threshold = 1
+            else:
+                area_threshold = int(layer_config)
```

Comment:
Why not spell this `area_threshold = int(layer_config.get('area-inclusion-threshold', 1))`?